src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -330,18 +330,11 @@ const parseObjectToMongoObjectForCreate = (className, restCreate, schema) => {
 // Main exposed method to help update old objects.
 const transformUpdate = (className, restUpdate, parseFormatSchema) => {
   let mongoUpdate = {};
-  let acl = addLegacyACL(restUpdate)._acl;
```

Comment:
If you are suggesting not changing line 333 then none of the if clauses will execute on lines 336, 339, 342

The object returned by addLegacyACL has the format:
```
{
 _acl: {
   id: w,
   id: r},
 _rperm: [],
_wperm: []
}
```